src/plugins/intel_gpu/tests/shape_infer/pad_si_test.cpp

```diff
@@ -42,8 +42,10 @@ TEST_P(pad_test_three_input, shape_infer) {
     auto pads_begin_prim = std::make_shared<input_layout>("pads_begin", p.pads_begin_layout);
     auto pads_end_prim = std::make_shared<input_layout>("pads_end", p.pads_end_layout);
 
-    auto border_prim = std::make_shared<border>("output", input_info("input"),
-                                                input_info("pads_begin"), input_info("pads_end"),
+    int32_t non_constant_input_mask = border::PAD_NON_CONST_INPUT::BEGIN | border::PAD_NON_CONST_INPUT::END;
+    auto border_prim = std::make_shared<border>("output", std::vector<input_info>{input_info("input"), input_info("pads_begin"), input_info("pads_end")},
+                                                non_constant_input_mask,
+                                                std::vector<int64_t>{}, std::vector<int64_t>{},
```

Comment:
You replaced existing constant input tests to non const input test. Could you please make it to be tested for both cases? (const / non const begin/end)

Reply:
This test was originally non-const input test. And there is a constant test after this test, "pad_test_single_input"